src/interfaces/AuthProvider.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
why?  can you disable individual lines with the proper offending rule?  Or, can you disable the rules your concerned with here, or is there something more fundamental?  Its really good to be able to see the reason or rule in the comment....

i.e. `/* eslint-disable object-curly-spacing */`

which lets the reader know that object-curly-spacing is the rule that is an issue.